### ISSUE TITLE
[BUG] opml import from akregator shows no new entries

### ISSUE THREAD

**Issue by swestlake** [marienfressinaud/FreshRSS#3967]
**Describe the bug**
no new entries get shown after importing an opml file
(export was done from akregator)

**Environment information (please complete the following information):**
- FreshRSS version: 1.18.1
- PHP version: 7.4.24
- Installation type: docker

**Comment by Alkarex** [comment]
Could you please provide a sample of your OPML file reproducing the problem?

**Comment by swestlake** [comment]
```
<?xml version="1.0" encoding="UTF-8"?>
<opml version="1.0">
 <head>
  <text/>
 </head>
 <body>
  <outline archiveMode="globalDefault" version="RSS" text="StackFlow - Linux" description="most recent 30 from stackoverflow.com" id="2144086840" useCustomFetchInterval="false" copyright="" type="rss" xmlUrl="https://stackoverflow.com/feeds/tag/linux" comment="" maxArticleNumber="1000" title="StackFlow - Linux" fetchInterval="-1" htmlUrl="https://stackoverflow.com/questions/tagged/?tagnames=linux&amp;sort=active" maxArticleAge="60"/>
 </body>
</opml>
```

I removed a local file:// resource which I thought could be causing the importing issue.
`faviconUrl="file:///home/_user/.cache/favicons/cdn.sstatic.net_Sites_stackoverflow_Img_favicon.png"`
(which was going after the title property)

so I tried the import with and without faviconUrl and results are no different.

this opml file now only has one entry  -- not sure if count# was an issue, so even down to one rss feed confirms there is a problem with the syntax somewhere.

thanks

**Comment by swestlake** [comment]
the format of opml v1 exported from akregator is not accepted by freshrss, though opml v2 is understood by it.

on a recent test, I imported akregator's v1 opml format into another rss client(rssguard), and from there I exported the feeds into a newly created opml v2 file.

I imported the new opml v2 file into freshrss and all feeds seemed to import correctly.

so freshrss supports opml v2 correctly, just not the default v1 format by which akregator gets set to.
(akregator unfortunately doesn't export to opml v2, only to v1)
